### PR TITLE
Show the probe family in `oscap --v`

### LINKED ISSUE #650: Include schema in supported OVAL objects list of `oscap --version`
When printing capabilities of oscap, all OVAL objects are printed as names, without schema prefix. This is fine for experienced user, but for newcomer, who want to write some rule, full name would be easier to find in documentation.

Example:
Instead of `selinuxboolean`, `linux:selinuxboolean` would be printed.

### PR DESCRIPTION
**Pull request by mpreisler**
Fixes #650 

Example:
```
==== Supported OVAL objects and associated OpenSCAP probes ====
(null)        system_info                  probe_system_info           
independent   family                       probe_family                
independent   filehash                     probe_filehash              
independent   environmentvariable          probe_environmentvariable   
independent   textfilecontent54            probe_textfilecontent54     
independent   textfilecontent              probe_textfilecontent       
independent   variable                     probe_variable              
independent   xmlfilecontent               probe_xmlfilecontent        
independent   environmentvariable58        probe_environmentvariable58 
independent   filehash58                   probe_filehash58            
linux         dpkginfo                     probe_dpkginfo              
linux         inetlisteningservers         probe_inetlisteningservers  
linux         rpminfo                      probe_rpminfo               
linux         partition                    probe_partition             
linux         iflisteners                  probe_iflisteners           
linux         rpmverify                    probe_rpmverify             
linux         rpmverifyfile                probe_rpmverifyfile         
linux         rpmverifypackage             probe_rpmverifypackage      
linux         selinuxboolean               probe_selinuxboolean        
linux         selinuxsecuritycontext       probe_selinuxsecuritycontext
linux         systemdunitproperty          probe_systemdunitproperty   
linux         systemdunitdependency        probe_systemdunitdependency 
unix          file                         probe_file                  
unix          interface                    probe_interface             
unix          password                     probe_password              
unix          process                      probe_process               
unix          runlevel                     probe_runlevel              
unix          shadow                       probe_shadow                
unix          uname                        probe_uname                 
unix          xinetd                       probe_xinetd                
unix          sysctl                       probe_sysctl                
unix          gconf                        probe_gconf                 
unix          routingtable                 probe_routingtable          
unix          symlink                      probe_symlink    
```

### CODE REVIEW

**Comment by rsprudencio** [comment]
@mpreisler Do you think it's worth to set probe_system_info to "oval_system_characteristics" object family? Everything else looks great to me.

File: utils/oscap.c [hunk 1]
```diff
@@ -181,7 +181,7 @@ static int print_versions(const struct oscap_action *action)
 	printf("\n");
 
 	printf("==== Supported OVAL objects and associated OpenSCAP probes ====\n");
```
Comment (yuumasato) [inline utils/oscap.c]: I think adding a column description, similar to a table header, like:
```
Oval Schema           Oval object            OpenSCAP Probe
```
would add value for newcomers.

Comment (yuumasato) [inline utils/oscap.c]: Maybe `Oval Family` would be more appropriate.

**Comment by mpreisler** [comment]
Implemented the header suggestion from @yuumasato .

Example:
```
==== Supported OVAL objects and associated OpenSCAP probes ====
OVAL family   OVAL object                  OpenSCAP probe              
----------    ----------                   ----------                  
(null)        system_info                  probe_system_info           
independent   family                       probe_family                
independent   filehash                     probe_filehash              
independent   environmentvariable          probe_environmentvariable   
independent   textfilecontent54            probe_textfilecontent54     
independent   textfilecontent              probe_textfilecontent       
...
```

**Comment by mpreisler** [comment]
> @mpreisler Do you think it's worth to set probe_system_info to "oval_system_characteristics" object family? Everything else looks great to me.

system_info is kind of special and I think us showing `(null)` there is fine.

**Comment by jan-cerny** [comment]
@mpreisler 
> system_info is kind of special and I think us showing (null) there is fine.
Agreed.

**Comment by jan-cerny** [comment]
I have tested and checked this PR. LGTM.

**Pull request merged**